spec/migrations/20190531161732_add_guid_to_miq_databases_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_migration
+
+describe AddGuidToMiqDatabases do
+  let(:miq_database) { migration_stub(:MiqDatabase) }
+
+  migration_context :up do
+    it "sets miq_database guids" do
+      obj1 = miq_database.create!
+
+      expect(obj1.reload.guid).to eq(nil)
```

Comment:
@Fryguy before the migration `#guid` doesn't exist

Reply:
Yeah I removed that...seems I didn't push it 😩